content/docs/glossary/decentralized-exchange.mdx

```diff
@@ -3,41 +3,42 @@ title: Decentralized Exchange
+When creating a "buy"/"bid" order in Stellar via the [Manage Buy Offer](../start/list-of-operations.mdx#manage-buy-offer) operation, the price is specified as 1 unit of the base currency (the asset being bought), in terms of the quote asset (the asset that is being sold). For example, if you're _buying_ 100 XLM in exchange for 20 USD, you would specify the price as `{20, 100}`, which would be the equivalent of 5 XLM for 1 USD (or $.20 per XLM).
 
-When creating a "sell"/"offer"/"ask" offer in Stellar via the [Manage Sell Offer](../start/list-of-operations.mdx#manage-sell-offer) operation, the price is specified as 1 unit of what you're _selling_ in terms of what you're _buying_. For example, if you're _selling_ 10 USD in exchange for 100 XLM, you would specify the price as `{100, 10}`, which would be the equivalent of 1 USD @ 10 XLM (_nice profit_).
+When creating a "sell"/"offer"/"ask" order in Stellar via the [Manage Sell Offer](../start/list-of-operations.mdx#manage-sell-offer) operation, the price is specified as 1 unit of base currency (the asset being sold), in terms of the quote asset (the asset that is being bought). For example, if you're _selling_ 100 XLM in exchange for 40 USD, you would specify the price as `{40, 100}`, which would be the equivalent of 2.5 XLM for 1 USD (or $.40 per XLM) (_nice profit_).
 
 #### Fees
 
-It's important to note that the price you set is unrelated to the fee you pay for submitting the offer as a part of a transaction. Fees are always paid in the native currency of the network (lumens), and are related to the transaction that you submit to the network (which contains your offer operation) as opposed to your offer itself.
+It's important to note that the price you set is unrelated to the fee you pay for submitting the order as a part of a transaction. Fees are always paid in the native currency of the network (lumens), and are related to the transaction that you submit to the network (which contains your order operation) as opposed to your order itself.
 
 For more information, take a look at [our guide on fees in Stellar](./fees.mdx).
 
-## Passive Offers
+## Passive Order
 
-**Passive offers** allow markets to have zero spread. If you want to offer USD from anchor A for USD from anchor B at a 1:1 price, you can create two passive offers so the two offers don't fill each other.
+**Passive orders** allow markets to have zero spread. If you want to exchange USD from anchor A for USD from anchor B at a 1:1 price, you can create two passive orders so the two orders don't fill each other. (will this actually be a zero spread? it should still be a tick wide right? or would it be a "locked market", where the bid and the ask is at the same price?)
```

Comment:
Flagging the last sentence. It looks like a comment, rather than an edit. We should make sure to resolve this question and not push that to `master`.

I don't know the answer, but it seems like it could be any of the above depending on the orders. The documentation wording seems deliberately vague, to just illustrate that this is a possibility - rather than saying "this is how you do it."

Reply:
Let's clarify this...see the comment for line #39... I'm not sure what happens with locked/crossed markets in Stellar. Might be easier to hop on call with anyone else necessary to go over this.

Reply:
If you create two passive offers at the same price, it will actually be zero spread. This is the only way to get a zero spread market in Stellar. (I'd need to double check, but it might also be possible to get a zero spread market by modifying an offer that was created passive. Either way, this is conceptually the same behavior.)